src/services/pi_stats/mount_health.py

```
@@ -0,0 +1,212 @@
+        identifier = item.get("id")
+        path = item.get("path")
+        source = item.get("source")
+        required = item.get("required_directories", [])
+        if not isinstance(identifier, str) or not MOUNT_ID.fullmatch(identifier):
+            raise ValueError(
+                f"Mount check {index} id must match {MOUNT_ID.pattern!r}",
+            )
+        if identifier in identifiers:
+            raise ValueError(f"Duplicate mount check id: {identifier!r}")
+        if not isinstance(path, str) or not PurePosixPath(path).is_absolute():
+            raise ValueError(f"Mount check {identifier!r} path must be absolute")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)